app/models/invoice.rb

```diff
@@ -1,2 +1,8 @@
 class Invoice < ApplicationRecord
+  belongs_to :customer
+  has_many :transactions
+
+  def self.pending_invoices
+    where(status: 'in progress').order(:created_at)
+  end
```

Comment:
looking at the wording of this user story, i think the goal here is to join invoices with invoice items where the status is not shipped (select for the status of the invoice items, not the status of the invoice)